src/main/java/pages/cartPage/ShoppingCartTable.java

```diff
@@ -0,0 +1,282 @@
+package pages.cartPage;
+
+import enums.Currency;
+import lombok.Getter;
+import lombok.Setter;
+import models.ProductModel;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+import static engine.drivers.WebDriverFactory.getWebDriverInstance;
+
+@Getter
+@Setter
+public class ShoppingCartTable extends BaseTable {
+    protected WebDriver driver = getWebDriverInstance();
+    public By cartEmptyMessagePanel = By.xpath("//div[@class='contentpanel1']");
+    public By deleteItemButton = By.xpath(".//i[contains(@class, 'fa-trash-o')]/parent::a");
+    public By quantityItemInput = By.xpath(".//input[contains(@id, 'cart_quantity')]");
+    public By cartUpdateButton = By.id("cart_update");
+    public By cartCheckoutButton = By.id("cart_checkout1");
+
+    public ShoppingCartTable() {
```

Comment:
I think this class would look way better I you will move all the methods about row management to separate class called ShoppingCartTableRow :)

Reply:
Done
```public class ShoppingCartTableRow {}``` was created